e2e/playwright.config.ts

```diff
@@ -0,0 +1,86 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+let htmlReporter = 'html';
+let baseURL = 'http://localhost:8000';
+if ( process.env.TAQASTA_E2E_IN_DOCKER ) {
+  htmlReporter = [ [ 'html', { open: 'never'}] ];
+  baseURL = 'http://web:80/';
+}
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: './tests',
+  /* Run tests in files in parallel */
+  fullyParallel: false,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: (process.env.CI || process.env.TAQASTA_E2E_IN_DOCKER) ? 1 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : 1,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: htmlReporter,
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    baseURL: baseURL,
+    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+    trace: 'on-first-retry',
+  },
+
+  // global timeout per suite
+  globalTimeout: 60 * 60 * 1000,
+
```

Comment:
We can add `timeout` here and remove it from `defineConfig` on each of the test suites